test/concurrent.cc

```diff
@@ -77,36 +139,36 @@ TEST(ConcurrentTest, Success) {
     callback();
   }
 
-  EXPECT_THAT(future.get(), UnorderedElementsAre("1", "2"));
+  this->ExpectElementsAre(std::move(future), "1", "2");
```

Comment:
I'd like to see this as:
```suggestion
  EXPECT_THAT(future.get(), this->OrderedOrUnorderedElementsAre("1", "2"));
```